development/java/Rubberduck.Parsing/Grammar/build.gradle

```diff
@@ -0,0 +1,52 @@
+buildscript {
+    repositories {
+        mavenCentral();
+    }
+
+    dependencies {
+        classpath group: 'org.antlr', name: 'antlr4', version: '4.7.2'
+    }
+}
+
+plugins {
+    id 'java'
+}
+
+group 'com.rubberduckvba.rubberduck.parsing'
+version '1'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    compile group: 'org.antlr', name: 'antlr4-runtime', version: '4.7.2'
+}
+
+def grammerCodeGenDest = "${projectDir}/src/main/gen"
```

Comment:
typo: `grammer` should be `grammar`

Reply:
@Vogel612
I will fix this in the next commit.